schemas/get-worker-type-response.yml

```diff
@@ -0,0 +1,112 @@
+id:       http://schemas.taskcluster.net/aws-provisioner/v1/get-worker-type-response.json#
```

Comment:
Instead of calling this file: `get-worker-type-response.json` how about `worker-type.json`.

The create/get requests can both use the same schema. And if you need to embed in another schema you can use `$ref` construct... Or the special `contants.js` file... There is no need for duplication of JSON schemas.. We have plenty of tools make it generic.